cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -34,16 +34,41 @@ class ActOnStabilizerCHFormArgs:
     Measurements are currently not supported on this object.
```

Comment:
Measurements are supported from this point onwards! :)

Reply:
So true!